modules/csg/csg_shape.cpp

```diff
@@ -292,11 +323,10 @@ static void _pack_manifold(
 	// runIndex needs an explicit end value.
 	mesh.runIndex.push_back(mesh.triVerts.size());
 	ERR_FAIL_COND_MSG(mesh.vertProperties.size() % mesh.numProp != 0, "Invalid vertex properties size.");
-	mesh.Merge();
-	r_manifold = manifold::Manifold(mesh);
-	manifold::Manifold::Error err = r_manifold.Status();
-	if (err != manifold::Manifold::Error::NoError) {
-		print_error(String("Manifold creation from mesh failed:" + itos((int)err)));
+	if (mesh.Merge()) {
+		r_manifold = manifold::Manifold(mesh);
+	} else {
+		r_manifold = manifold::Manifold();
```

Comment:
Hmm, I think you were checking for errors properly before. If `mesh.Merge()` is false, all that means is it didn't change anything because the mesh is already manifold - probably not a good time to return something empty.

Reply:
I restored the old version with some bits that state the node path of the CSG node.